### PR TITLE
Hardware updates

### PR DESCRIPTION
**Pull request by RealOrangeOne**
- Improve the content in power and motor board docs.
- Structure placeholder pages for future content
- Add battery documentation

#### Todo
- [x] Charging
- [x] Pi
- [x] Sprinkle more warning messages in battery docs, can't be too careful!

### CODE REVIEW

File: content/kit/batteries/_index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- If a battery has any cuts, nicks, exposed copper on wires or is bulging to the point of no longer being squishy, contact us immediately.
+
+{{% notice warning %}}
+Disobeying the above warnings greatly increases the risk of damage to your battery, which can have fatal consequences. If you have any questions or doubts, contact us immediately. 
+{{% /notice %}}
+
+## Storing Batteries
+When your batteries are not actively in use, they should be safely stored. You must disconnect the batteries from all electrical equipment, and place them in the battery charging bag. You should then store the charging bag in a safe location.
+
+If you are not going to use the batteries for a long period of time, such as more than a week, then you should charge the batteries before hand. By themselves, batteries will discharge very slowly, and over-discharging will lead to damage. We recommend that you charge all batteries to at least 70% of capacity before being stored for a long period.
```
Comment (kierdavis) [inline content/kit/batteries/_index.md]: Is it worth mentioning the "storage charge" mode of the LiPo charger here?

File: content/kit/batteries/charging.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+4. Place the battery in the charging bag.
+5. Configure the charger using the menu system as follows. 
+
+### Starting Charging
+
+{{% notice warning %}}
+Do not attempt to charge your battery if any of the cables are lose or frayed. Contact us immediately if you think there's an issue!
+{{% /notice %}}
+
+1. Press `Start/Enter` to select lithium battery charging.
```
Comment (kierdavis) [inline content/kit/batteries/charging.md]: This should say "LiPo battery charging", as many chargers (including the IMAX B6s we provide to teams) support other types of lithium battery (e.g. Li-ion, LiFe).

File: content/kit/batteries/charging.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+{{% notice warning %}}
+The important thing to note here is that the number of cells detected by the charger (_R_) and the number of cells set by you (_S_) are both 3. *If either of these numbers are different immediately power down the charger and contact us.*
+{{% /notice %}}
+
+The screen shows the charging current, the current voltage, the time elapsed and the energy provided to the battery.
+
+### Stopping Charging
+The charger will beep when the battery is fully charged. You must then:
+
+1. Press `Stop` button. The beeping should now stop.
```
Comment (kierdavis) [inline content/kit/batteries/charging.md]: Technically this isn't true - our chargers will continue to beep even after pressing stop. I think the docs should match how charging actually works as closely as possible to reduce unnecessary worry.

Comment (RealOrangeOne) [reply content/kit/batteries/charging.md]: I've not seen this? Once they start beeping, pressing the button stops them? From what I remember anyway

Comment (kierdavis) [reply content/kit/batteries/charging.md]: I thought they carried on beeping until they'd completed their full "beep duration" - could be wrong though. I'll check it tonight if I get time.

Comment (RealOrangeOne) [reply content/kit/batteries/charging.md]: After testing, it does stop it

**Pull request merged**